src/alpine/3.6-ForHelix/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM alpine:3.6
+
+RUN apk update
+
+RUN apk add --no-cache \
+        autoconf \
+        bash \
+        build-base \
+        clang \
+        clang-dev \
+        cmake \
+        coreutils \
+        curl-dev \
+        gcc \
+        gettext-dev \
+        git \
+        icu-dev \
+        krb5-dev \
+        libtool \
+        libunwind-dev \
+        libffi \
+        linux-headers \
+        llvm \
+        make \
+        openssl \
+        openssl-dev \
+        paxctl \
+        py-cffi \
+        python \
+        python-dev \
+        sudo \
+        util-linux-dev \
+        zlib-dev
+
+RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/main add --no-cache \
+        userspace-rcu-dev \
+        lttng-ust-dev
+
+RUN apk -X https://dl-cdn.alpinelinux.org/alpine/edge/testing add --no-cache \
+        lldb-dev
+
+RUN /usr/sbin/adduser -D -G adm -s /bin/sh helixbot
+
+# Restore pip and set environment
+ADD https://bootstrap.pypa.io/get-pip.py /home/helixbot/get-pip.py
```

Comment:
It is a Docker best practice to not use ADD.  The preferred pattern for this scenario would be to download the file via something like wget, run the python script and then remove the file all within one instruction to avoid bloating the resulting image.

Reply:
I see, sounds good I'll do that.